tasks/dashboard.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # tasks/dashboard.yml: Deployment of the Uchiwa dashboard
-  - include: "{{ ansible_distribution }}/dashboard.yml"
+  - include: "{{ role_path }}/tasks/{{ ansible_distribution }}/dashboard.yml"
```

Comment:
This provides the ability for other roles to include this role

https://github.com/ansible/ansible/issues/20589